source/identity-access-management/roles.rst

```diff
@@ -117,6 +117,213 @@ manage their own account information. This role cannot view, create or destroy
 project resources and it does not permit the uploading of SSH keys or the
 viewing of project usage and quota information.
 
+More information
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. raw:: html
+
+   <details>
+   <summary>For a more comprehensive list of the exact permissions that each
+   role gives, you can <a>click here</a> to view a full list for the
+   non-kubernetes roles.</summary>
+
+.. code-block:: console
+
+   +-----------------------+------------------------------------------------------------------------+
+   | Role                  | Permissions                                                            |
+   +=======================+========================================================================+
+   | Project Member        | ALARM SERVICE                                                          |
```

Comment:
Project Member is kind of a name or description of the role. We'd better use the real role name in Keystone (as you did for k8s roles).

Reply:
Form the dashboard, the users will see "Project Member" not _member_ as we've formatted it this way on the dash. But we have not formatted the Kubernetes roles, hence the difference in names here.